src/njoy21/input/LEAPR/Card5/Iel/test/Iel.test.cpp

```diff
@@ -0,0 +1,37 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "LEAPR Card5 iel input values", 
+  "[LEAPR], [Card5], [Iel]"){
+  GIVEN( "valid iel input values" ){
+    WHEN( "iel input provided" ){
+      std::vector<int> validValues{0, 1, 5, 6};
```

Comment:
Since the valid values are discrete (and not a lot), we should explicitly test each valid value. That way we know if we inadvertently exclude one.